tests/ip-fragment/bench.c

```diff
@@ -84,6 +87,72 @@ static void test_benchmark_ip_fragment(void)
 	pg_brick_destroy(ip_fragment);
 }
 
+static void test_benchmark_ip_defragment(int mtu)
+{
+	struct pg_error *error = NULL;
+	struct pg_brick *ip_fragment;
+	struct pg_brick *collect;
+	struct pg_bench bench;
+	struct ether_addr mac = {{0x52,0x54,0x00,0x12,0x34,0x11}};
+	uint32_t len;
+	struct pg_bench_stats stats;
+	int pkt_len = 1500;
+	struct rte_mbuf **tmp_pkts;
+	uint64_t tmp_mask = 1;
+
+	pg_bench_init(&bench);
+	ip_fragment = pg_ip_fragment_new("ip_fragment",
+					 WEST_SIDE, mtu, &error);
+	g_assert(!error);
+	collect = pg_collect_new("col_west", 1, 1, &error);
+	g_assert(!error);
+
+	pg_brick_link(collect, ip_fragment, &error);
+	g_assert(!error);
+
+	tmp_pkts = pg_packets_create(tmp_mask);
+	tmp_pkts = pg_packets_append_ether(
+		tmp_pkts,
+		tmp_mask,
+		&mac, &mac,
+		ETHER_TYPE_IPv4);
+
+	/* tmp_post_burst_op = append_blank; */
+	len = sizeof(struct ipv4_hdr) + sizeof(struct udp_hdr) + pkt_len;
+	pg_packets_append_ipv4(
+		tmp_pkts,
+		tmp_mask,
+		0x000000EE, 0x000000CC, len, 17);
+	tmp_pkts = pg_packets_append_blank(tmp_pkts, tmp_mask,
+					   pkt_len);
+	PG_FOREACH_BIT(tmp_mask, i) {
+		struct eth_ipv4_hdr *pkt_buf =
+			rte_pktmbuf_mtod(tmp_pkts[i], struct eth_ipv4_hdr *);
+		pkt_buf->ip.fragment_offset = 0;
+	}
+
+	pg_brick_burst_to_west(ip_fragment, 0, tmp_pkts, tmp_mask, &error);
+	bench.pkts = pg_brick_east_burst_get(collect, &bench.pkts_mask, &error);
+	pg_packets_incref(bench.pkts, bench.pkts_mask ^ 1);
+
+	bench.input_brick = ip_fragment;
+	bench.input_side = WEST_SIDE;
+	bench.output_brick = ip_fragment;
+	bench.output_side = EAST_SIDE;
+	bench.output_poll = false;
+	bench.max_burst_cnt = 1000000;
```

Comment:
You should add this option in function parameters.
- Some bench are very quick so you should make this value higher.
- Some bench are too slow so you should make this value lower.